web_src/js/components/RepoContributors.vue

```diff
@@ -88,6 +91,9 @@ export default {
   methods: {
     sortContributors() {
       const contributors = this.filterContributorWeeksByDateRange();
+      const min = dayjs(this.xAxisMin).format('YYYY-MM-DD');
+      const max = dayjs(this.xAxisMax).format('YYYY-MM-DD');
+      this.searchQuery = `${this.repoLink}/commits/branch/${this.repoBranch}/search?q=after:${min}, before:${max}, author:`;
```

Comment:
```suggestion
      this.searchQuery = `${this.repoLink}/commits/branch/${this.repoBranch}/search?q=after:${min},before:${max},author:`;
```
also, it looks really weird to construct the search query like this.
Wouldn't it make more sense to have `?after=…&before=…&author=…`?
And is all that functionality already supported?
I would have guessed that you need to add it yourself…

Reply:
Ideally we should use `URLSearchParams` to construct search params:

```js
String(new URLSearchParams({foo: "bar", baz: "2"})) // 'foo=bar&baz=2'
```

Reply:
>Wouldn't it make more sense to have ?after=…&before=…&author=…?

yeah, it would. But I can't do that way because it is not something I decided. Gitea currently works the other way.

>And is all that functionality already supported?

Yes. And it won't work if I accept your code suggestion. The spaces are important. As I said, this is how it works.

Reply:
If it's existing API, I guess this weird format is okay, otherwise it would be a breaking change. Still we should ensure the individual paramters and path segments are encoded correctly.